### ISSUE TITLE
Zopim Chat button stops users from submitting forms on Mobile

### ISSUE THREAD

**Issue by paddyow** [Kademi/kademi-dev#193]

**Comment by paddyow** [comment]
Solution: Hide chat button on mobile devices:


<script type=”text/javascript”>
var ua = navigator.userAgent.toLowerCase(),
platform = navigator.platform.toLowerCase();
platformName = ua.match(/ip(?:ad|od|hone)/) ? ‘ios’ : (ua.match(/(?:webos|android)/) || platform.match(/mac|win|linux/) || [‘other’])[0],
isMobile = /ios|android|webos/.test(platformName);
if (!isMobile) {
//place the Zopim chat widget code here without the <script> tags
}
</script>



![screenshot_2015-02-23-21-09-12](https://cloud.githubusercontent.com/assets/2201340/6326910/a77427fc-bbaa-11e4-8384-24ad485094d2.png)

**Comment by skwidge** [comment]
Zopim provide a solution for hiding the button on mobile devices on their blog. I've just implemented this and pushed it up to the master repository on kademi-dev;

	modified:   src/main/webapp/templates/apps/admin/login.html
	modified:   src/main/webapp/templates/themes/admin2/normal.html

This is a fairly safe change, the worst that can happen is that the live chat button might be hidden when you don't want it to be. I've tested it using different user agents in chrome.
